core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -355,6 +356,8 @@ object TopicCommand extends Logging {
 
     val forceOpt = parser.accepts("force", "Suppress console prompts")
 
+    val excludeInternalTopic = parser.accepts("exclude-internal", "exclude the intenral topics when lising the topic name. the internal topic will be listed by default")
```

Comment:
I believe this flag does not make sense for commands other than --describe and --list (because they require --topic anyway ). Could you please also add a check to checkArgs() ?

Reply:
copy that.